src/Uno.UI.RemoteControl/Uno.UI.RemoteControl.Wasm.csproj

```diff
@@ -17,7 +17,7 @@
 
 		<Nullable>enable</Nullable>
 
-		<UnoRuntimeIdentifier Condition="'$(TargetFramework)'=='netstandard2.0'">WebAssembly</UnoRuntimeIdentifier>
+		<UnoRuntimeIdentifier>WebAssembly</UnoRuntimeIdentifier>
```

Comment:
This is a behavioral change again. Previously `UnoRuntimeIdentifier` is not set for this project.